lib/kubeclient/inflections.rb

```diff
@@ -0,0 +1,4 @@
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.irregular 'resource_quota', 'resource_quotas'
+  inflect.irregular 'resourcequota', 'resourcequotas'
+end
```

Comment:
I *think* you can just do `inflect.irregular 'quota', 'quotas'`, and it'll handle it as a suffix.

But either way, yes, this looks right.

The only potential problem, is that the library is now registering custom pluralizations which could affect the surrounding application -- that's unlikely to be something a developer expects when adding your gem to their Gemfile. I don't know of any better solution for that, though.